bindata/oauth-apiserver/deploy.yaml

```diff
@@ -101,18 +101,36 @@ spec:
         - mountPath: /var/log/oauth-apiserver
           name: audit-dir
         livenessProbe:
-          initialDelaySeconds: 30
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
+          initialDelaySeconds: 0
+          periodSeconds: 10
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 3
         readinessProbe:
-          failureThreshold: 10
           httpGet:
             scheme: HTTPS
             port: 8443
             path: readyz
-      terminationGracePeriodSeconds: 70 # a bit more than the 60 seconds timeout of non-long-running requests
+          initialDelaySeconds: 0
+          periodSeconds: 5
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 1
+        startupProbe:
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            path: healthz
+          initialDelaySeconds: 0
+          periodSeconds: 5
+          timeoutSeconds: 1
+          successThreshold: 1
+          failureThreshold: 30
+      terminationGracePeriodSeconds: 90 # a bit more than the 60 seconds timeout of non-long-running requests + the shutdown delay
```

Comment:
where does the referenced 60 seconds come from?

Reply:
> where does the referenced 60 seconds come from?

hardcoded (I think) into the apiserver for shutting down long running requests .